projects/storefrontlib/cms-components/navigation/scroll-to-top/scroll-to-top.component.ts

```diff
@@ -7,41 +7,48 @@
 import {
   ChangeDetectionStrategy,
   Component,
+  ElementRef,
   HostBinding,
   HostListener,
   OnInit,
+  Optional,
+  ViewChild,
+  inject,
 } from '@angular/core';
 import {
-  WindowRef,
   CmsScrollToTopComponent,
+  FeatureConfigService,
   ScrollBehavior,
+  WindowRef,
 } from '@spartacus/core';
 import { take } from 'rxjs/operators';
 import { CmsComponentData } from '../../../cms-structure/page/model/cms-component-data';
-import { ICON_TYPE } from '../../misc/icon/icon.model';
 import { SelectFocusUtility } from '../../../layout/a11y/index';
+import { ICON_TYPE } from '../../misc/icon/icon.model';
 
 @Component({
   selector: 'cx-scroll-to-top',
   templateUrl: './scroll-to-top.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ScrollToTopComponent implements OnInit {
-  iconTypes = ICON_TYPE;
+  readonly iconTypes = ICON_TYPE;
 
   @HostBinding('class.display')
-  display: boolean | undefined;
+  display: boolean = false;
```

Comment:
Lets avoid changing the type here so that `display` can still be `undefined` which can be treated as `falsey`.

Reply:
Change Reverted